lib/ui/text/paragraph_impl_blink.cc

```diff
@@ -0,0 +1,187 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/lib/ui/text/paragraph_impl_blink.h"
+#include "flutter/lib/ui/text/paragraph.h"
+#include "flutter/lib/ui/text/paragraph_impl.h"
+
+#include "flutter/common/threads.h"
+#include "flutter/sky/engine/core/rendering/PaintInfo.h"
+#include "flutter/sky/engine/core/rendering/RenderParagraph.h"
+#include "flutter/sky/engine/core/rendering/RenderText.h"
+#include "flutter/sky/engine/core/rendering/style/RenderStyle.h"
+#include "flutter/sky/engine/platform/fonts/FontCache.h"
+#include "flutter/sky/engine/platform/graphics/GraphicsContext.h"
+#include "flutter/sky/engine/platform/text/TextBoundaries.h"
+#include "lib/ftl/tasks/task_runner.h"
+#include "lib/tonic/converter/dart_converter.h"
+#include "lib/tonic/dart_args.h"
+#include "lib/tonic/dart_binding_macros.h"
+#include "lib/tonic/dart_library_natives.h"
+
+using tonic::ToDart;
+
+namespace blink {
+
+IMPLEMENT_WRAPPERTYPEINFO(ui, ParagraphImplBlink);
```

Comment:
This shouldn't need a WRAPPERTYPEINFO.  The Paragraph is the object with the Dart wrapper, not this object.